packages/rest/src/parser.ts

```diff
@@ -15,19 +15,10 @@ import {
 import {ResolvedRoute} from './router/routing-table';
 type HttpError = HttpErrors.HttpError;
 
-type jsonBodyFn = (
-  req: ServerRequest,
-  cb: (err?: Error, body?: {}) => void,
-) => void;
-const jsonBody: jsonBodyFn = require('body/json');
-
-// tslint:disable:no-any
+// tslint:disable-next-line:no-any
 type MaybeBody = any | undefined;
-// tslint:enable:no-any
 
-const parseJsonBody: (req: ServerRequest) => Promise<MaybeBody> = promisify(
-  jsonBody,
-);
+const jsonBodyAsync = promisify(require('body/json'));
```

Comment:
Please preserve the name `parseJsonBody`, method/function names should start with a verb.

Also please verify that the compiler is correctly inferring the type of `promisify(require('body/json'))` and does not fall back to `any`. If it does not, then you need to preserve some of the type definitions above.